peleffy/utils/input.py

```diff
@@ -0,0 +1,190 @@
+"""
+This module contains a set of classes and methods designed to handle
+input data.
+"""
+from peleffy.utils import Logger
+
+
+class PDB(object):
```

Comment:
Class name might be too ambiguous (I know it was me the one who suggested it). I would change it for `PDBFile`, for instance (or any other that you might think of).